modules/renter/repair.go

```diff
@@ -13,13 +14,22 @@ import (
 )
 
 const (
-	// When a file contract is within this many blocks of expiring, the renter
-	// will attempt to renew the contract.
-	renewThreshold = 2000
-
 	hostTimeout = 15 * time.Second
 )
 
+// When a file contract is within 'renewThreshold' blocks of expiring, the renter
+// will attempt to renew the contract.
+var renewThreshold = func() types.BlockHeight {
+	switch build.Release {
+	case "testing":
+		return 20
+	case "dev":
+		return 200
+	default:
+		return 2000
+	}
+}()
```

Comment:
I guess it's strange that you can have a variable that's a function which gets called.